web/e2e/driver/driver.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+
+import { type WebDriver, Builder } from 'selenium-webdriver'
+import fetch from 'node-fetch'
+import childProcess from 'child_process'
+import serverConfigs from '../config/configs'
+import { clone } from 'lodash'
+import type { EndToEndConfigType } from '../config/configs'
+
+const BROWSERSTACK_EXHAUSTED_MESSAGE = 'All parallel tests are currently in use, including the queued tests. ' +
```

Comment:
Should we include browserstack stuff here? I suspect that no one will execute this with browserstack.

Reply:
Maybe it is cool for safari, IE etc.
Can we maybe refactor this to share the code with native?